keps/sig-release/2853-kubernetes-kubernetes-repository-branch-rename/kep.yaml

```diff
@@ -0,0 +1,54 @@
+title: Changing kubernetes/kubernetes default branch name to main
+kep-number: 2853
+authors:
+  - "@cpanato"
+owning-sig: sig-release
+participating-sigs:
```

Comment:
Please include sig-testing, as this will certainly impact CI.

Reply:
cc also @spiffxp @stevekuznetsov

Reply:
I would think also SIG Contributor Experience?

Reply:
added the sigs